tao_compiler/mlir/disc/transforms/conv_rewriter.cc

```diff
@@ -168,7 +168,8 @@ LogicalResult inferExpectedLayout(ConvParams& params) {
   outputLayout.resize(rank);
 
   if (onGpu) {
-    if (inputTy.getElementType().isF16() && filterTy.getElementType().isF16()) {
+    if (cc_major >= 8 ||
```

Comment:
what if TF32 is disabled on device with `cc_major >= 8`

Reply:
will this affect `-multi-cc` ?

Reply:
> what if TF32 is disabled on device with cc_major >= 8

Ok, I added a check for NVIDIA_TF32_OVERRIDE.

> will this affect -multi-cc ?

In multi-cc mode, disc compiler tries to get optimal performance for the current architecture it executes on. For other architectures, functional correctness is guaranteed.